index.js

```diff
@@ -6,6 +6,7 @@ const morgan = require("morgan");
 
 const userRouter = require("./routers/userRouter");
 const postRouter = require("./routers/postRouter");
+const tagRouter = require("./routers/tagRouter");
```

Comment:
Your `index.js` is very clean Angela. Using this pattern is a great way to just keep your resources straight, and make handling more complex resources more manageable 🔥